src/sentry/issues/search.py

```diff
@@ -246,22 +246,15 @@ def _query_params_for_generic(
     return None
 
 
-def get_search_strategies(
-    organization: Organization, actor: Optional[Any]
-) -> Mapping[int, GroupSearchStrategy]:
+def get_search_strategies() -> Mapping[int, GroupSearchStrategy]:
     strategies = {}
     for group_category in GroupCategory:
         if group_category == GroupCategory.ERROR:
             strategy = _query_params_for_error
         elif group_category == GroupCategory.PERFORMANCE:
-            if not features.has(
-                "organizations:issue-platform-search-perf-issues", organization, actor=actor
-            ):
-                strategy = _query_params_for_perf
-            else:
-                strategy = functools.partial(
-                    _query_params_for_generic, categories=[GroupCategory.PERFORMANCE]
-                )
+            strategy = functools.partial(
+                _query_params_for_generic, categories=[GroupCategory.PERFORMANCE]
+            )
```

Comment:
Ideally we consolidate this in the future so that all non error categories are grouped and make just a single query, rather than query per category